cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,38 @@
-/// <reference types='cypress' />
+/// <reference types="cypress" />
 
-describe('Book Store app', () => {
+describe('Book Store App', () => {
   before(() => {
-    
+    cy.viewport(1920, 1080);
+    cy.visit('https://demoqa.com/login');
   });
 
-  it('', () => {
-    
+  beforeEach(() => {
+    cy.reload();
+  });
```

Comment:
you have only one `it` so you can use only one before

Reply:
DOne